packages/wp-story-editor/src/index.js

```diff
@@ -47,14 +47,19 @@ import {
   MediaUpload,
 } from './components';
 import getApiCallbacks from './api/utils/getApiCallbacks';
+import { transformGetStoryResponse } from './api/utils';
+
+window.webStories = window.webStories || {};
+window.webStories.domReady = domReady;
 
 /**
  * Initializes the web stories editor.
  *
- * @param {string} id       ID of the root element to render the screen in.
- * @param {Object} config   Story editor settings.
+ * @param {string} id           ID of the root element to render the screen in.
+ * @param {Object} config       Story editor settings.
+ * @param {Object} initialEdits Initial edits.
  */
-const initialize = (id, config) => {
+window.webStories.initializeStoryEditor = (id, config, initialEdits) => {
```

Comment:
We might want to do the same for the dashboard in the future, for consistency.

Reply:
Absolutely!